static/app/components/growingInput.tsx

```diff
@@ -47,9 +47,26 @@ export const GrowingInput = forwardRef<HTMLInputElement, InputProps>(
 
     // If the input is controlled we resize it when the value prop changes
     useLayoutEffect(() => {
+      // This mutationObserver is used to resize the input if the inputRef's style changes
+      // It's necessary because the refs do not trigger useEffects
+      const observer = new MutationObserver(() => {
+        inputRef.current && resize(inputRef.current);
+      });
+
       if (isControlled && inputRef.current) {
         resize(inputRef.current);
       }
+
+      if (inputRef.current) {
+        observer.observe(inputRef.current, {
+          attributes: true,
+          attributeFilter: ['style'],
```

Comment:
This seems like it would probably result in the width being calculated multiple times, since the resize function will modify the `width` style